docs/examples/notebooks/02-set_neuron_params.ipynb

```diff
@@ -0,0 +1,281 @@
+    "plt.title(\"Probed Results (Dimension 1: sin)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 0])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 0])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\")\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "plt.title(\"Probed Results (Dimension 2: cos)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 1])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 1])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the output for both dimensions is zero, as expected."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Set Weights and Test\n",
+    "\n",
+    "Now we will use the values we generated in [Step 3](#Step-3:-Create-Some-Weights) in our FPGA network and test again to show the desired neuron parameters have been set correctly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # Explicitly set ensemble attributes\n",
+    "    fpga_ens.ensemble.bias = bias\n",
+    "    fpga_ens.ensemble.encoders = encoders\n",
+    "    fpga_ens.ensemble.gain = gain\n",
+    "\n",
+    "    # To set the decoders, we need to use nengo.NoSolver\n",
+    "    # to tell the builder we will provide out own decoders\n",
+    "    # (you can also set \"weights\" here instead,\n",
+    "    # see nengo.solvers.NoSolver for more info)\n",
+    "    fpga_ens.connection.solver = NoSolver(decoders.T)  # Transposed\n",
+    "\n",
+    "with nengo_fpga.Simulator(model) as sim:\n",
+    "    sim.run(2)  # Run for 2 seconds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(16, 5))\n",
+    "plt.subplot(1, 2, 1)\n",
+    "plt.title(\"Probed Results (Dimension 1: sin)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 0])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 0])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\")\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "plt.title(\"Probed Results (Dimension 2: cos)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 1])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 1])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we've set the neuron parameters to those we generated, we can see the FPGA ensemble successfully reproduces the sine and cosine signals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## More Information\n",
```

Comment:
I feel like this section should be at the very top of the notebook, just after the intro paragraph. I would take out the heading and move all of the bits in this section to before ## Step 1.